hw/ip/i2c/data/i2c.hjson

```diff
@@ -728,5 +736,37 @@
         { bits: "7:0" }
       ]
     }
+    { name: "HOST_NACK_HANDLER_TIMEOUT"
```

Comment:
In the message of the commit that contains this change, "This CSR controls a timeout mechanism where if SW does handle a pending 'nak'" should instead say "if SW does **not** handle a pending 'nak'", right?

Reply:
Indeed it should! ty